runtime/cudaq/qis/execution_manager.cpp

```diff
@@ -0,0 +1,21 @@
+/*************************************************************** -*- C++ -*- ***
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ *******************************************************************************/
```

Comment:
nit: This is the old 81 column style header. Please use the new one. (`.cpp` files should not have an Emacs mode tag.)